linode/linodego
# Fix deprecated comment; run `make tidy`; fix errors by `revive` linter; disable `perfsprint` linter

- Fix a format issue of the deprecation note, aligned with the [official doc](https://go.dev/wiki/Deprecated).
- `make tidy`
- Disable `perfsprint` linter because of a large size is required to fix it, e.g. replacing all `fmt.Sprintf("/some/api/%s", betaID)` with `"/some/api/" + betaID`.
- Fixed lint errors reported by `revive`.

## How to test
`go build` and `golangci-lint run` is sufficient to verify if these are good. And they ran by the GHA as well.
